src/utils/box.css

```diff
@@ -0,0 +1,6 @@
+.u-box {
+  border: 1px solid var(--color-border-dark);
```

Comment:
I'd prefer to see if we can also make variables of the borders including the sizes.

@xadamy Is that possible? This is kinda the reason I wanted to have the `@extend` feature in place. That way we can create utilities for shadows and borders like this: `.u-border--1`, `.u-border--2`, `.u-shadow--1` etc and use those in the components. Otherwise we end up with slightly different shadows all around.

Reply:
I resolved the shadow for now by moving it to a separate utility. We currently only have `u-shadow`. We can add more later.